rpc/wrtc_client.go

```diff
@@ -187,27 +190,12 @@ func dialWebRTC(
 	// bool representing whether initial sdp exchange has occurred
 	haveInit := false
 
-	errCh := make(chan error)
-	sendErr := func(err error) {
-		if haveInit && isEOF(err) {
-			logger.Warnf("caller swallowing err %v", err)
-			return
-		}
-		if s, ok := status.FromError(err); ok && strings.Contains(s.Message(), noActiveOfferStr) {
-			return
-		}
-		logger.Warnf("caller received err %v of type %T", err, err)
-		select {
-		case <-exchangeCtx.Done():
-		case errCh <- err:
-		}
-	}
 	var uuid string
 	// only send once since exchange may end or ICE may end
-	var sendDoneErrorOnce sync.Once
+	var sendDoneOnce sync.Once
 	sendDone := func() error {
 		var err error
-		sendDoneErrorOnce.Do(func() {
+		sendDoneOnce.Do(func() {
 			_, err = signalingClient.CallUpdate(exchangeCtx, &webrtcpb.CallUpdateRequest{
```

Comment:
Do we want to `Warn`-log any errors from this `CallUpdate` and have `sendDone` return void? I'm assuming you'll get rid of the "`sendDone` return value ignored" lint errors somehow but curious if you'll want to log the errors or literally ignore them. I would vote for the former.